src/react-components/styles/theme.js

```diff
@@ -34,6 +38,9 @@ export function useTheme(themeId) {
 
   useEffect(
     () => {
+      // Make sure themes is properly configured. (It must be an array.)
+      if (!themes || !themes.find || !themes.__proto__ !== Array.prototype) return;
```

Comment:
There is an `Array.isArray()` function in JS to do this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

`if (!themes || !Array.isArray(themes)) return;`